tests/mbt/model/ccv_model.qnt

```diff
@@ -948,4 +1060,163 @@ module ccv_model {
                 VotingPowerChange("node1", 50)
             }
         )
+
+    // ======= tests for consumer initiated slashing =======
+
+    run DoubleDowntimeSlashTest = 
```

Comment:
Nit: For what "DoubleDowntime" stands for?

Reply:
Good call, at some point this tested two downtime slashes in the same trace, but was changed. I'll rename it